SmartDeviceLink-iOS/SmartDeviceLink/SDLTCPTransport.m

```diff
@@ -63,8 +63,10 @@ - (void)connect {
 
 - (void)sendData:(NSData *)msgBytes {
     dispatch_async(_sendQueue, ^{
-        NSString *byteStr = [SDLHexUtility getHexString:msgBytes];
-        [SDLDebugTool logInfo:[NSString stringWithFormat:@"Sent %lu bytes: %@", (unsigned long)msgBytes.length, byteStr] withType:SDLDebugType_Transport_TCP toOutput:SDLDebugOutput_DeviceConsole];
+        @autoreleasepool {
+            NSString *byteStr = [SDLHexUtility getHexString:msgBytes];
+            [SDLDebugTool logInfo:[NSString stringWithFormat:@"Sent %lu bytes: %@", (unsigned long)msgBytes.length, byteStr] withType:SDLDebugType_Transport_TCP toOutput:SDLDebugOutput_DeviceConsole];
+        }
```

Comment:
Since this method is passing this block to a separate thread, it's in apple's recommendations that if you're doing lots of local variable creation you should `@autoreleasepool`. This is only a problem if you call it a lot like we often might. It may result in a slight performance hit to use it, but since we are already using it, we may as well clean up all the temp variables created in the block rather than just some and leaving others to be cleaned up later.